Lambdas/GHHooks/Extensions/+PR.Label.swift

```diff
@@ -1,38 +1,38 @@
 import GitHubAPI
 
 extension PullRequest {
-    var knownLabels: [KnownLabel] {
-        self.labels.compactMap {
-            KnownLabel(rawValue: $0.name)
+
+    enum KnownLabel: String {
+        case semVerMajor = "semver-major"
+        case semVerMinor = "semver-minor"
+        case semVerPatch = "semver-patch"
```

Comment:
Fabian added a label `semver-noop` on PostgresNIO to mean the same thing we use `no-release-needed` for; in case we ever add any behavior for it (such as release notes in the style Fabian uses), we should probably recognize it as an alias.